src/main.cpp

```diff
@@ -45,6 +45,7 @@
 #include "config_compp.h"
 #include "config_effects.h"
 #include "lvl_script.h"
+#include "lvl_script_conditions.h"
```

Comment:
why was this include added, it isn't used

Reply:
I merged something into this PR, then had difficulty building it.